roxie/ccd/ccdstate.cpp

```diff
@@ -2320,6 +2320,11 @@ class CRoxiePackageSetManager : public CInterface, implements IRoxieQueryPackage
                 miscDebugTraceLevel = control->getPropInt("@level", 0);
                 topology->setPropInt("@miscDebugTraceLevel", miscDebugTraceLevel);
             }
+            else if (stricmp(queryName, "control:multicast")==0)
```

Comment:
This isn't going to have any effect unless the Roxie is restarted. I'd be inclined to delete this bit of code.